delete-user-data/test-data/lib/index.js

```diff
@@ -0,0 +1,113 @@
+"use strict";
```

Comment:
This file shouldn't be committed, the previous one didn't commit this, only the `.ts` file